internal/generated/ubicloud_client/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
folder `internal/generated` has this content, as not only `ubicloud_client` is generated but also the models of data sources and resources:
```sh
❯ ls internal/generated
datasource_firewall       datasource_postgres       datasource_project        resource_firewall         resource_postgres         resource_project          ubicloud_client
datasource_firewall_rule  datasource_private_subnet datasource_vm             resource_firewall_rule    resource_private_subnet   resource_vm
```

Reply:
I guess I was just trying to do the minimum to make the makefile work, but I guess it does result in inconsistent treatment of ignored files in parent directories.